script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -136,8 +136,7 @@ I'm terrified at the prospect of what might happen next.
 [sha:7ac03a672799a4cd09140558df1671d1076db7be]{
 -- Page 525, Offset 10006.
 -- いま俺が生きているのは<た|・><ま|・><た|・><ま|・>だ。
-// ruby dots <た|・><ま|・><た|・><ま|・>
-It's only by chance that I'm still alive.
+It's only %{i}by chance%{/i} that I'm still alive.
```

Comment:
Not sure I love this one either

Reply:
again, I think if _just_ "chance" is italicized, it's a positive change